package/gluon-config-mode-outdoor/luasrc/lib/gluon/config-mode/wizard/0250-outdoor.lua

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
This Option is still redundant. There should be only one checkbox in the config mode. Maybe you can remove this file or replace it with a banner like the autoupdater banner to inform the user wether the outdoor mode is active or not.

Reply:
Ok, so than maybe you can remove the config inside the "advanced mode" or at leas the redundant checkboxes.

Reply:
The rationale is:

- Enable outdoor mode automatically when the model is primarily used outdoors
- Inform users in wizard when outdoor mode is enabled and allow them to disable it
- Allow advanced users to enable outdoor mode, even when it wasn't autodetected
- Allow advanced users to configure the HT mode